src/ui/public/index_patterns/_get_computed_fields.js

```diff
@@ -6,7 +6,7 @@ export default function () {
   let scriptFields = {};
   let docvalueFields = [];
 
-  docvalueFields = _.pluck(self.fields.byType.date, 'name');
+  docvalueFields = _.pluck(_.reject(self.fields.byType.date, 'scripted'), 'name');
```

Comment:
FWIW, plucks deprecated in `lodash` from v4 onwards, so if we ever upgrade, we'll have to remove usage. (use 'map' instead).